Gemfile

```diff
@@ -35,6 +37,8 @@ gem 'font-awesome-rails'
 gem 'font-awesome-sass'
 # sass mixins that makes you happy
 gem 'bourbon'
+# normalize!
+gem 'normalize-rails'
```

Comment:
Normalize resets the css, so that any Browser-specific special default css styles is removed.